pkg/framework/plugins/defaultevictor/defaultevictor.go

```diff
@@ -58,10 +59,20 @@ func IsPodEvictableBasedOnPriority(pod *v1.Pod, priority int32) bool {
 	return pod.Spec.Priority == nil || *pod.Spec.Priority < priority
 }
 
-// HaveEvictAnnotation checks if the pod have evict annotation
-func HaveEvictAnnotation(pod *v1.Pod) bool {
-	_, found := pod.ObjectMeta.Annotations[evictPodAnnotationKey]
-	return found
+// HaveEvictAnnotation checks if the pod have evict annotation and its value
+func HaveEvictAnnotation(pod *v1.Pod) (bool, bool) {
+	svalue, found := pod.ObjectMeta.Annotations[evictPodAnnotationKey]
```

Comment:
thoughts on using `strconv.ParseBool`?

Reply:
I think it depends on how much we want to push in terms of backward compatibility: the current code is checking only the presence of the annotation.
`strconv.ParseBool` will return a `syntaxError` for values (like an empty string) that fails to be cleanly converted to a Bool. Using `strconv.ParseBool`  and catching the error to consider it as a True to mimic backward compatibility is also an option.

Reply:
Yes, this is also a great suggestion.

Reply:
Done